bin/setup

```diff
@@ -14,17 +14,20 @@ FileUtils.chdir APP_ROOT do
   # Add necessary setup steps to this file.
 
   puts '== Installing dependencies =='
-  system! 'gem install bundler --conservative'
+  system! 'gem install bundler:2.0.1'
+  system! 'bundler config github.https true'
   system('bundle check') || system!('bundle install')
```

Comment:
Should probably use `bundle install -j$(nproc)` as above to take advantage of parallelism.